feat: homepage, quiz-selection, rules

ParthParolekar opened the pull request:
Added homepage and quiz selection page

ParthParolekar commented:
![image](https://user-images.githubusercontent.com/69164301/153567938-cc9b8c0d-6148-450d-970d-ef287ecd5f84.png)

![image](https://user-images.githubusercontent.com/69164301/153568003-23c24482-87fa-4bcd-a58f-8a27bae54cbe.png)

![image](https://user-images.githubusercontent.com/69164301/153568057-aa408ea8-59f6-4d4a-a1eb-f5f26d81e423.png)

partha8 reviewed (approved):
looks good

anik31 commented on pages/category/category.html:
```diff
@@ -0,0 +1,87 @@
+      integrity="sha512-BnbUDfEUfV0Slx6TunuB042k9tuKe3xrD6q4mg5Ed72LTgzDIcLPxg6yI2gcMFRyomt+yJJxE+zJwNmxki6/RA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Chooes Quiz</title>
+  </head>
```
typo

anik31 commented:
The select another quiz button is not readable. It may have low color contrast, you can check it out.

ParthParolekar commented:
> The select another quiz button is not readable. It may have low color contrast, you can check it out.

Changed the button and fixed the typo! Thank you for pointing it out!

Developers-codz reviewed (approved):
Looks Good

ParthParolekar commented:
Thank you @anik31 @Developers-codz @partha8  and @VLeads  for the reviews. Merging now
